packages/deploy/src/core/CoreContracts.ts

```diff
@@ -771,163 +769,189 @@ export default class EvmCoreDeploy extends AbstractCoreDeploy<config.EvmCoreCont
+        for (const watcher of remoteConfig.configuration.watchers) {
+          const watcherPermission =
+            await this.xAppConnectionManager.watcherPermission(
+              watcher,
+              remoteDomainNumber,
+            );
+          checklist.equals(
+            true,
+            watcherPermission,
+            `Watcher permission for ${remoteDomain} is set`,
           );
-        checklist.equals(true, watcherPermission, 'Watcher permission');
+        }
       }
     }
 
     //  ========= GovRouter =========
     // GovernanceRouter upgrade setup contracts are defined
     checklist.assertBeaconProxy(
       this.data.governanceRouter,
-      'Governance router',
+      'Governance router is deployed',
     );
     // localDomain
     const govLocalDomain = await this.governanceRouter.localDomain();
-    checklist.equals(this.domainNumber, govLocalDomain, 'Gov domain');
+    checklist.equals(
+      this.domainNumber,
+      govLocalDomain,
+      'GovernanceRouter localDomain is properly configured',
+    );
     // recoveryTimelock
     const recoveryTimelock = await this.governanceRouter.recoveryTimelock();
     checklist.equals(
       ethers.BigNumber.from(
         domainConfig.configuration.governance.recoveryTimelock,
       ).toHexString(),
       recoveryTimelock.toHexString(),
-      'Recovery time lock',
+      'GovernanceRouter Recovery time lock is properly configured',
     );
     // recoveryActiveAt
     const recoveryActiveAt = await this.governanceRouter.recoveryActiveAt();
-    checklist.equals(true, recoveryActiveAt.eq(0), 'Recovery activeAt');
+    checklist.equals(
+      true,
+      recoveryActiveAt.eq(0),
+      'GovernanceRouter Recovery is not initiated',
+    );
     // recoveryManager
     const recoveryManager = await this.governanceRouter.recoveryManager();
     checklist.equalIds(
       domainConfig.configuration.governance.recoveryManager,
       recoveryManager,
-      'Recovery Manager',
+      'GovernanceRouter Recovery Manager is properly configured',
     );
     // governor
-    const govId = await this.governanceRouter.governor();
-    checklist.exists(this.context.protocol, 'Protocol config');
+    checklist.exists(this.context.protocol, 'Protocol config exists');
     if (this.context.protocol) {
+      const govId = await this.governanceRouter.governor();
       const expectedGovId = isGovernor
         ? this.context.protocol.governor.id
         : ethers.constants.AddressZero;
-
-      checklist.equalIds(expectedGovId, govId, 'Gov address');
+      checklist.equalIds(expectedGovId, govId, 'Governor address is set');
 
       const govDomain = await this.governanceRouter.governorDomain();
       const expectedGovDomain = this.context.protocol.governor.domain;
-      checklist.equals(expectedGovDomain, govDomain, 'Gov domain');
+      checklist.equals(expectedGovDomain, govDomain, 'Governor domain is set');
```

Comment:
Perhaps being visually explicit?
```suggestion
      checklist.equals(expectedGovDomain, govDomain, 'Governor domain is set to ${govDomain}`);
```

Reply:
Governor domain is ${govDomain}, expected ${expectedGovDomain}